v63004/instream.csh

```diff
@@ -25,13 +25,14 @@
 # gtm1042	    [jake]  Tests the that env variable gtm_mstack_size sets the size of the M stack as expected
 # gtm8891	    [vinay] Tests that <side-effect-expression><pure-Boolean-operator>$SELECT(0:side-effect-expression)) sequence produces a SELECTFALSE runtime error
 # gtm8894	    [vinay] Tests that $zreldate outputs in the form YYYYMMDD 24:60
+# gtm8643	    [jake]  Tests that YDB no longer enforces queue depth limit of 5 for SOCKET devices. NOT TESTED YET (Confirms connection attempts fail upon transient issues)
 
```

Comment:
I've tried to specify in the instream.csh comment that the latter portion of the subtest hasn't been made yet. I figured I should make you aware of this, as you may find it unnecessary or undesirable.

Reply:
Thanks. That helps. I will look into it to see if the rest is easily testable or not.

Reply:
I took a look at this and was not able to come up with a simple test for the retry issue the release note talks about. So please add a note that only the first sentence in the release note is tested.